axelor-project/src/main/resources/views/Project.xml

```diff
@@ -158,16 +158,29 @@
 		                   	<span>{{record.totalRealHrs}} </span><span x-translate>hours</span>
 		       			]]></viewer>
 					</field>
-					<button name="planPlanningBtn" title="Plan project" css="btn-custom text-left" onClick="save,action-project-view-project-planning-wizard"  hideIf="$popup()" colSpan="3"/>
+					
+					<button name="planPlanningBtn" title="Plan project" css="btn-custom text-left" onClick="save,action-project-view-project-planning-wizard"  hideIf="$popup()" colSpan="3"/>				
 					<button name="addMultipleLinesBtn" title="Add multiple lines" css="btn-custom text-left" onClick="save,action-project-view-add-multiple-project-planning-time" colSpan="3" hideIf="$popup()"/>
-					<button name="addProjectPlanningTimeLineBtn" title="New line"  css="btn-custom text-left" colSpan="3" onClick="save,action-project-view-project-planning-time-add-new" />
-					<button name="removeProjectPlanningTimeLineBtn" title="Remove lines" css="btn-custom text-left" colSpan="3" onClick="save,action-project-attrs-project-planning-time-remove" />
+					<button name="addProjectPlanningTimeLineBtn" title="New line"  css="btn-custom text-left" colSpan="3" onClick="save,action-project-view-project-planning-time-add-new,save" />
+					<button name="removeProjectPlanningTimeLineBtn" title="Remove lines" css="btn-custom text-left" colSpan="3" onClick="save,action-project-attrs-project-planning-time-remove,save" />
+					<panel-dashlet name="projectPlanningTimePanel" title="Project planning time" action="action-project-dashlet-project-planning-time" colSpan="12" canSearch="true"/>
 					<field name="$projectPlanningTimeSet" hidden="true" type="many-to-many" widget="TagSelect" title="Project Planning Time"
 					colSpan="12" target="com.axelor.apps.project.db.ProjectPlanningTime" canEdit="false" canNew="false" 
 					grid-view="project-planning-time-grid" form-view="project-planning-time-form"/>
 					<button name="removeSelectedPlanningTimeBtn" colSpan="4" hidden="true"  title="Remove selected lines"  onClick="action-project-validate-remove-project-planning-time,action-project-planning-time-method-remove-project-planning-time"/>
 					<button name="cancelRemovePlanningTimeBtn" colSpan="4" hidden="true" title="Cancel" onClick="action-project-attrs-project-planning-time-cancel-remove" />
-					<panel-dashlet name="projectPlanningPanel" title="Project planning time" action="action-project-dashlet-project-planning-time" colSpan="12" canSearch="true"/>
```

Comment:
Position of 'projectPlanningTimeSet' was before 'projectPlanningPanel' , but with your change its changed now its after dashlet. When user wants to remove he/she has to check below dashlet now. 
Change it to and follow this pattern for all new remove buttons.